static/js/addons/cl.utils.js

```diff
@@ -49,8 +49,12 @@ var Cl = window.Cl || {};
         },
 
         // INFO: simple mobile and tablet filter that can be extended
-        mobile: function () { return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i)); },
-        tablet: function () { return (navigator.userAgent.match(/iPad/i)); }
+        mobile: function () {
+            return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i));
```

Comment:
I think that since multi-device is now becoming a requirement, it will be good to offer a standard way to deal with this. But we can take our time to figure out what's the best solution here.